hack/update-generated-protobuf-dockerized.sh

```diff
@@ -86,6 +86,7 @@ PACKAGES=(
   k8s.io/apiserver/pkg/apis/audit/v1beta1
   k8s.io/apiserver/pkg/apis/audit/v1
   k8s.io/apiserver/pkg/apis/example2/v1
+  k8s.io/api/auditregistration/v1alpha1
```

Comment:
order

Reply:
this fails if the apiserver types aren't there first as it relies on the Policy type